### PR TITLE
update osmnx code

### PR DESCRIPTION
**Pull request by martinfleis**
OSMnx needs to reproject the graph before calling `ox.consolidate_intersections` for tolerance to work properly.

Also, `ox.simplify_graph` just removes nodes of a degree 2, which is likely pointless for what you're trying to do here.

### CODE REVIEW

File: paper.qmd [hunk 1]
```diff
@@ -190,11 +190,12 @@ import osmnx as ox
 G = ox.graph_from_place("Edinburgh Waverley", network_type="walk", buffer_dist=500)
 # Plot the graph in an interactive map:
 # ox.plot_graph_folium(G)
+# project to 27700
+G_projected = ox.project_graph(G, 27700)
```
Comment (Robinlovelace) [inline paper.qmd:194]: Quick question on this: would

```suggestion
G_projected = ox.project_graph(G, 'EPSG:27700')
```

work here?

I think it's good practice to include the context: more searchable and there are different CRS databases than EPSG, with ESRI having its own system if I remember correctly. I will give it a go to find out!

File: paper.qmd [hunk 2]
```diff
@@ -190,11 +190,12 @@ import osmnx as ox
...
 # simplify:
-G_simplified = ox.consolidate_intersections(G, tolerance=0.0002, rebuild_graph=True)
-ox.plot_graph_folium(G_simplified)
-G_simplified_graph = ox.simplify_graph(G, tolerance=0.0002, rebuild_graph=True)
-
+G_simplified = ox.consolidate_intersections(G_projected, tolerance=10, rebuild_graph=True)
```
Comment (Robinlovelace) [inline paper.qmd:196]: +1 to having that in meters and not meaningless units of degrees!

**Review by Robinlovelace — APPROVED** [review]

File: paper.qmd (hunk 1 above)
Comment (martinfleis) [reply paper.qmd:194]: Yes, it should. I was just lazy :)

Comment (Robinlovelace) [reply paper.qmd:194]: Done in latest commit to main : )

**Pull request merged**